Editor/Bootstrap/BootstrapSettings.cs

```diff
@@ -117,30 +118,35 @@ public void SelectBucket(string name)
             _status.IsDisplayed &= string.IsNullOrEmpty(BucketName);
         }
 
-        public void CreateBucket()
+        public Response CreateBucket()
         {
             if (!CanCreate)
             {
-                return;
+                return new Response()
+                {
+                    ErrorMessage = CannotCreateErrorMessage
+                };
```

Comment:
Why do we need to return the response now? I see that we have a success handler `OnBucketCreated` and a failure handler `OnBucketCreationFailure`. Is it not sufficient to call `OnBucketCreationFailure` with this response?

Reply:
This was so we can determine whether to show the error to the user within the UI. I will include the normal bucket failure procedure as well.